docs/index.md

```
@@ -23,276 +23,127 @@ search, or large volumes of structured and semi-structured data, CrateDB gives
-:class-title: sd-fs-5
-
-Explore the many options for integrating with third-party applications and
-frameworks.
-:::
-
-::::::
+> When using CrateDB, a project that got started around the same time, it's like
+you've stumbled into an alternative reality where Elastic is a proper database.
+>
+> <small>-– Henrik Ingo, Nyrkiö Oy, independent database consultant, MongoDB</small>
 
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)